spec/unit/recipes/default_spec.rb

```diff
@@ -0,0 +1,92 @@
+#
+# Cookbook:: bluematador_agent
+# Spec:: default
+#
+# Copyright:: 2018, The Authors, All Rights Reserved.
+
+require 'spec_helper'
+
+describe 'bluematador_agent::default' do
+
+  debian_runners = [
+    { platform: 'ubuntu', version: '16.04' },
```

Comment:
test 14.04? still under LTS

Reply:
Turns out this is less of an actual test and more of a "would the thing theoretically run".  Mostly just protects what we do in the case statements for different platforms...